homeassistant/components/amcrest/camera.py

```diff
@@ -51,6 +53,28 @@
 _SRV_TOUR_ON = "start_tour"
 _SRV_TOUR_OFF = "stop_tour"
 
+_SRV_PTZ_CTRL = "ptz_control"
+_ATTR_PTZ_TT = "travel_time"
+_ATTR_PTZ_MOV = "movement"
+_MOV = [
+    "zoom_in",
+    "zoom_out",
```

Comment:
These two are backwards. The first (that goes with `ZoomWide`) should be `zoom_out`, and the second (that goes with `ZoomTele`) should be `zoom_in`.

Reply:
fixed